BotDatabase.py

```diff
@@ -80,30 +90,48 @@ def CleanupBackups(self):
                 
     ### Adding/Updating/Removing Server Entries ###
     def AddBotGuilds(self, ListOwnerAndServerTuples, BotID:int):
-        BotAdditionUpdates = []
+        BotAdditionUpdates:list[Server] = []
         for Entry in ListOwnerAndServerTuples:
-            BotAdditionUpdates.append(Entry + (-1, 0, BotID))
+            server = Server(
+                bot_instance_id = BotID,
+                discord_server_id = Entry.id,
+                owner_discord_user_id = Entry.owner_id,
+                activator_discord_user_id = Entry.owner_id
```

Comment:
This should probably be -1

Reply:
will do

Reply:
Initial entries to the database are now -1 for activator id